services/kubernetes-dashboard/5.1.1/kubernetes-dashboard.yaml

```diff
@@ -77,4 +77,4 @@ data:
   name: "Kubernetes"
   dashboardLink: "/dkp/kubernetes/"
   docsLink: "https://kubernetes.io/docs/home/"
-  version: "5.1.1"
+  version: "2.4.0"
```

Comment:
just for my reference, this is the app version right?

Reply:
yep, app version (vs chart version which we also use as our "kommander application version" 😬 )

Reply:
cool cool thanks!